### PR TITLE
Added option to replace MiNNLO muR/muF binned variations with continuous polynomial variations

### PR DESCRIPTION
**Pull request by dbruschi**
This PR adds the option to replace MiNNLO muR/muF binned variations with continuous polynomial variations as in the theory agnostic implementation by the ERC ASYMOW group. To do this you run setupCombine.py with the "--muRmuFPolVar" option. This is implemented both the W mass analysis and the W-like analysis, and it doesn't require to define an out-of-acceptance dataset.

### CODE REVIEW

File: scripts/histmakers/mw_with_mu_eta_pt.py [hunk 1]
```diff
@@ -37,6 +37,9 @@
 parser.add_argument("--selectNonPromptFromSV", action="store_true", help="Test: define a non-prompt muon enriched control region")
 parser.add_argument("--selectNonPromptFromLighMesonDecay", action="store_true", help="Test: define a non-prompt muon enriched control region with muons from light meson decays")
 parser.add_argument("--useGlobalOrTrackerVeto", action="store_true", help="Use global-or-tracker veto definition and scale factors instead of global only")
+parser.add_argument("--muRmuFPolVarFilePath", type=str, default=f"{data_dir}/MiNNLOmuRmuFPolVar/", help="Path where input files are stored")
+parser.add_argument("--muRmuFPolVarFileTag", type=str, default="x0p50_y4p00_ConstrPol5Ext_Trad", choices=["x0p50_y4p00_ConstrPol5Ext_Trad"],help="Tag for input files")
```
Comment (bendavid) [inline scripts/histmakers/mw_with_mu_eta_pt.py:41]: These should be added to utilities/common.py rather than definiing them for each histmaker.

**Comment by bendavid** [comment]
Does using ```--muRmuFPolVar``` with setupCombine automatically disable the binned muR/muF variations?

**Comment by dbruschi** [comment]
> Does using `--muRmuFPolVar` with setupCombine automatically disable the binned muR/muF variations?

Yes. It's here: https://github.com/WMass/WRemnants/pull/470/files#diff-a965cc836621a611774f0f83fcf392697c103c5caacb506fafe455358dc690c4R143

**Comment by bendavid** [comment]
We should wait to merge this until it's understood whether it's affected by the issue in https://github.com/WMass/WRemnants/pull/472  (and it will need to be updated to a newer wremnants-data commit in any case)

**Comment by bendavid** [comment]
So I think this one is good to go once the wremnants-data commit is updated (should be to the current HEAD c04dcc6cfc69dbc4c3ef5c05c44de28d5526f6ff )

**Comment by dbruschi** [comment]
It doesn't seem to work

**Comment by dbruschi** [comment]
Actually let me redo it

**Pull request merged**